packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1 +1,15 @@
-//Styles for off-canvas editor, remove this line when you add some css to this file!
+.offcanvas-editor-list-view-tree-wrapper {
+	max-width: 100%;
+	overflow-x: auto;
+	.block-editor-list-view-block__menu-cell:last-child {
+		padding-right: 0;
+	}
+
+	.block-editor-list-view-leaf {
```

Comment:
Question: do we want to rely on the cascade or should we apply offcavas specific classnames to these elements and target them?

Why? Because in future if we want to re-merge into main list view we might find it easier to replace the offcanvas specific classes with more generic list view class variations. This way it will be easier to identify where the branches in visual styling occur.

I'm not saying I'm right on this but just raising a potential area for discussion (cc @draganescu @scruffian @ajlende ).

Reply:
I would. But I'm curious to know what other contributors think. This is just my thoughts...

Reply:
I agree with @getdave!

Reply:
Yes, 

> Keeping classes as close to the point at which we are branching the visual styling (from the base list view) it will make it easier to see where the changes occur. 

this is the gist.

Reply:
🥳